dist/js/selectize.js

```diff
@@ -2316,7 +2324,9 @@ $.extend(Selectize.prototype, {
 			}
 
 			self.$input.find(old.join(', ')).remove();
-			self.$input.append(fresh.join(''));
+			$(fresh.join('')).each(function() {
```

Comment:
in case fresh contains several new options, but I'm not sure that this is possible, but the `fresh` is an array and who knows?